src/aioftp/client.py

```diff
@@ -501,9 +501,19 @@ def parse_list_line_windows(self, b):
         date_time_str = line[: date_time_end + 1].strip().split(" ")
         date_time_str = " ".join([x for x in date_time_str if len(x) > 0])
         line = line[date_time_end + 1 :].lstrip()
+
+        possible_formats = ["%m/%d/%Y %I:%M %p", "%m-%d-%y %I:%M%p"]
```

Comment:
I think this should bu multiline, so new rules can be added without commit history loss